core/deployment/src/main/java/io/quarkus/deployment/DevServices.java

```diff
@@ -0,0 +1,11 @@
+package io.quarkus.deployment;
+
+public interface DevServices {
+
+    /**
+     * Label to add to shared Dev Services running in containers.
+     * This allows other applications to discover the running service and use it instead of starting a new instance.
+     */
+    String DEV_SERVICE_LABEL = "quarkus-dev-service";
```

Comment:
I wonder if this should be specific to the devservices type, i.e. quarkus-dev-services-kafka. If everything uses the same name and we enabled shared support for other things you may end up with kafka connecting to postgresql if a user misconfigures something.

Reply:
Labels are key/value pairs, so while the key is the same, the value identifies the service. The lookup search for a container with the matching key/value and then verify that the expected port is mapper (exposed). 
We can have a specific one for each service, but we should make sure they start with the same prefix so we can identify the containers started by dev services.